### ISSUE TITLE
Tracking mdn/data deprecation

### ISSUE THREAD

**Issue by Rumyra** [mdn/mdn#166]

**Comment by lahmatiy** [comment]
I've been away from the discussion for a while, but I'd like to share my thoughts on the proposed deprecation of `mdn/data`. Personally, I believe this may not be the best course of action and here's why:
- `mdn/data` includes numerous proprietary definitions, properties, and syntax variants that are not present in CSS specs and may never be. However, these elements are still functional in modern browsers, ensuring web compatibility. Without these definitions, certain MDN articles could potentially malfunction (as I've noticed a some months ago, but glad to see they're functioning well now). This also affect tools that rely on `mdn/data`.
- `webref` primarily reflects the current state of the spec, which always has drafts and proposals for future syntaxes. Not all new entries into specs are maintained long-term. The implementation of new features in browsers is a gradual process. Some features might not even reach the stage of being a valid syntax. Hence, `compat-data` and `mdn-data` continue to be relevant.
- There are many syntaxes lacking formal definitions, with a significant amount of detail found in the prose of the spec. For instance, MDN's `<color>` [definition](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) offers modern syntaxes for `rgb()` and `hls()`, but lacks notes on "legacy" syntaxes. If the objective is to primarily teach new syntaxes, that's fine. However, references to legacy syntaxes should still exist for educational and explanatory purposes. Moreover, this situation is not suitable for linters and analytical tools that utilize `mdn/data` as a dictionary.
- It's not always clear which stage of CSS is represented by `webref` or similar resources. For some specs, we might need to reference a RC or REC, while others might require a working draft due to wide browser support (for instance Transform and Transition specs were in draft for a decade despite being in WD stage).

In conclusion, `mdn/data` serves as a human-curated CSS syntax dictionary that has been meticulously developed over the years to reflect the actual state of CSS. This makes it beneficial for reference, educational and tooling purposes (like validation and linting). In contrast, `webref` provides a snapshot of the current state of CSS specs, which can be beneficial for news, research, and synchronization with existing dictionaries. If the goal is to streamline synchronization, automation could be an option, but this process should still be moderated by humans, taking into consideration browser support.

**Comment by lahmatiy** [comment]
Btw, I've been working towards aligning `mdn/data` with `webref`. My initial step was to implement linting for parsability and connectivity of syntaxes. Unfortunately, the [PR](https://github.com/mdn/data/pull/597/) that I submitted last August is still awaiting review.

In the meantime, `mdn/data` currently has issues related to syntax connectivity (despite fixes in the PR):
<img width="878" alt="image" src="https://github.com/mdn/mdn/assets/270491/321b8379-4dfd-4dda-9fa4-b7a27472a3ea">

**Comment by Princejoroe** [comment]
love you all?
